lib/any.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var R = require('ramda');
+
+/**
+ * Output a block (or its inverse) based on whether or not any of the supplied
+ * arguments are truthy.
+ *
+ * @since v0.11.0
+ * @param {...*} values One or more values to test against.
+ * @param {Object} options
+ * @return {String|Boolean}
+ * @example
+ * var a = true;
+ * var b = 0;
+ * var c = false;
+ *
+ * {{#any a b}}✔︎{{else}}✘{{/all}} //=> ✔︎
+ * {{#any b c}}✔︎{{else}}✘{{/all}} //=> ✘
+ * 
+ * {{#if (any a b)}}
+ *   Also works inline!
+ * {{/if}}
+ */
+
+function any() {
+  var values = R.dropLast(1, arguments);
+  var options = R.last(arguments);
+  var result = false;
+  var i;
+
+  for (i = 0; i < values.length; i++) {
```

Comment:
@tylersticka Similar to [my other curiosity comment](https://github.com/cloudfour/core-hbs-helpers/pull/71/files#r308460629), could you use [`Array.prototype.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here instead of a `for` loop?

[`Array.prototype.some` also has good browser support](http://kangax.github.io/compat-table/es5/#test-Array_methods_Array.prototype.some_a_href=_https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some_title=_MDN_documentation_img_src=_../mdn.png_alt=_MDN_(Mozilla_Development_Network)_logo_width=_15_height=_13_/_/a_nbsp;). 🙂